### ISSUE TITLE
Adding a tutorial/minimal working example to help new users?

### ISSUE THREAD

**Issue by Speldosa** [openlilylib/snippets#152]
Would you consider creating a short tutorial or minimal working example for all us Lilypond n00bs out there (in my case, I still feel like a n00b although I've created several scores within Lilypond)? I've been really exited to try out the different styles (for example Improviso which looks absolutely stunning), but haven't been able to do it.

I think that there are a lot of potential users like me that are discourages from the library because of the rather high bar of entry, and a simple tutorial (going from downloading the library, placing it in a certain path, writing a minimal working example ly file, and horsing around with the --include commands in the terminal) would probably somewhat alleviate that problem. (I would be willing to contribute with such a tutorial myself if somebody just could give me a few pointers on how to do it.)

<img width="382" alt="screenshot 2016-09-25 16 17 17" src="https://cloud.githubusercontent.com/assets/1475822/18815831/ac1cc5e8-833b-11e6-9e56-c78130c21932.png">
<img width="728" alt="screenshot 2016-09-25 16 18 07" src="https://cloud.githubusercontent.com/assets/1475822/18815832/ac36d636-833b-11e6-996c-96731a99ca1e.png">

**Comment by PaulMorris** [comment]
Maybe adding such a tutorial to the material on the wiki?

https://github.com/openlilylib/snippets/wiki/Installation

-Paul

**Comment by Speldosa** [comment]
Sure, that would be one way of doing it (and then add a link to the wiki from the readme, saying that more detailed instructions can be found there).

**Comment by uliska** [comment]
There's one fundamental problem with the documentation: the annoying fact that much of openLilyLib is so temporary.

Specifically the "packages" below the `/ly` directory are actually all due to be moved to independent packages that work together with the [oll-core](https://github.com/openlilylib/oll-core) package. And I'm still waiting for the opportunity and spare time (haha) to think about and implement a _sustainable_ and comprehensive solution for documenting these packages and the openLilyLib system.

But as an interim solution (that may well be integrated in a future "proper" solution) I'm all for creating any kind of tutorial material. I see several options for this (and will gladly give a helping hand):
- Extending the Wiki as mentioned (maybe with adding a more explicit link on the main README page)
- Writing a tutorial on [Scores of Beauty](http://lilypondblog.org)
- Adding a section on the (awfully-WIP) ["LilyPond Book"](https://book.openlilylib.org/)
- Writing documentation section(s) as Markdown files in the actual repository (will be rendered in the Github interface)
- Writing something on https://openlilylib.org (requires knowledge in writing AngularJS  based web applications (actually: helping me getting on track again ;-) )
- Writing something as a GitBook, which can be uploaded somewhere on openlilylib.org

**Comment by Speldosa** [comment]
Yeah, it would probably be a good idea too start describing things at the highest level first. I've been looking around a bit in this whole openlilybib repository, and I can't really figure out what's what and how everything fits together. 

(I came here from [this blog post](http://lilypondblog.org/2016/01/arnold/) on Scores of Beauty, just because I wanted to use a different music font, and now I feel quite lost why I even needed to come here in the first place; why couldn't I simply go to the [improviso](https://github.com/OpenLilyPondFonts/improviso) page, download the font, and do... something?)
